src/BridgeMaker.js

```diff
@@ -1,13 +1,30 @@
 /**
  * 다리의 길이를 입력 받아서 다리를 생성해주는 역할을 한다.
  */
+//  제공된 BridgeMaker 객체를 활용해 구현해야 한다.
+//  BridgeMaker에 프로퍼티를 추가할 수 없다.
+//  BridgeMaker의 파일 경로는 변경할 수 없다.
+
+const { COMMAND } = require('./utils/constants/game');
+
+//  BridgeMaker의 메서드의 시그니처(인자, 이름)와 반환 타입은 변경할 수 없다.
 const BridgeMaker = {
   /**
    * @param {number} size 다리의 길이
    * @param {function(): number} generateRandomNumber 무작위 값을 생성해주는 함수
    * @return {string[]} 입력받은 길이에 해당하는 다리 모양. 위 칸이면 U, 아래 칸이면 D로 표현해야 한다.
    */
-  makeBridge(size, generateRandomNumber) {},
+  makeBridge(size, generateRandomNumber) {
+    const bridge = [];
+
+    for (let i = 0; i < size; i += 1) {
+      const randomNumber = generateRandomNumber();
+      if (randomNumber === 0) bridge.push(COMMAND.MOVE.DOWN);
+      else if (randomNumber === 1) bridge.push(COMMAND.MOVE.UP);
```

Comment:
프로그래밍 요구 사항에 `else`를 쓰는 것을 지양해야하는 조건이 있어서,
아래와 같이 쓰시는게 어떠실까요? 🤔

```suggestion
      const randomNumber = generateRandomNumber();
      bridge.push(
            randomNumber === 1
            ? COMMAND.MOVE.UP 
            : COMMAND.MOVE.DOWN 
      );
```

Reply:
오.. 3항연산자를 저렇게 괄호 내부에서 쓸수도 있겠네요. 저는 지금까지 `randomNumber === 1 ? bridge.push(COMMAND.MOVE.UP) : bridge.push(COMMAND.MOVE.DOWN) ` 항상 이렇게 써오다 보니 고정관념이 박혔던거 같아요! 감사합니다. 훨씬 보기 좋아졌네요
그리고 else if 문을 쓴건 0 또는 1의 값만 들어오는 걸 확실히 하고 싶어서 쓴거긴한데.. 다시 생각해보니 이 부분은 예외처리를 따로 해주는 식으로 해야할 것 같네요!